snooty/test_postprocess.py

```diff
@@ -2422,3 +2422,33 @@ def test_target_quotes() -> None:
     </target>
 </root>""",
         )
+
+
+def test_metadata() -> None:
+    with make_test(
+        {
+            Path(
+                "source/index.txt"
+            ): """
+======
+Test
+======
+
+This is a test intro
+            """,
+            Path(
+                "snooty.toml"
+            ): """
+name = "test_name"
+intersphinx = ["https://docs.mongodb.com/manual/objects.inv"]
```

Comment:
Do we need the intersphinx inventory for this test?

Reply:
Excellent! Intersphinx inventories involve doing a lot of work we don't really want to overdo in tests :D